docs/structured-streaming-programming-guide.md

```diff
@@ -510,8 +510,7 @@ returned by `SparkSession.readStream()`. In [R](api/R/read.stream.html), with th
 #### Input Sources
 There are a few built-in sources.
 
-  - **File source** - Reads files written in a directory as a stream of data. Supported file formats are text, csv, json, orc, parquet. See the docs of the DataStreamReader interface for a more up-to-date list, and supported options for each file format. Note that the files must be atomically placed in the given directory, which in most file systems, can be achieved by file move operations.
-
+  - **File source** - Reads files written in a directory as a stream of data. Files will be processed in the order of file modification time. If `latestFirst` is set, order will be reversed. Supported file formats are text, CSV, JSON, ORC, Parquet. See the docs of the DataStreamReader interface for a more up-to-date list, and supported options for each file format. Note that the files must be atomically placed in the given directory, which in most file systems, can be achieved by file move operations.
```

Comment:
why are ` text, CSV, JSON, ORC, Parquet` capitalized? I thought file format names are lower cased

Reply:
Maybe identifiers in code like `csv` would be lowercase, but these formats are acronyms and are properly capitalized when named. CSV = Comma Separated Values, etc.

Reply:
do we need to change this ?

Reply:
I don't believe so.